### DISCUSSION TITLE
Vamos mudar de assunto. O que você gosta além do código?

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1721] [Discussão]
A vida não pode ser monocromática.
Então o que vocês fazem no tempo livre? O que gostam de fazer fora código, trabalho. Não vale falar 'criar projetos próprio.'...

Eu me dedico a academia. Leitura de filosofia. Tentando aprender violão. Feriado perfeito é ir para praias de trilhas.

**Comment by viniman** [comment]
Eu gosto de ficar navegando na internet igual retardado buscando novos conhecimentos, no geral gosto de manter minha saúde com um futebolzinho de salão (saldosa pelada) no final de semana, correr pelo menos 3 vezes por semana, estudar investimento, marketing e desenvolvimento pessoal.

Voltei ao hobby de leitura, minha meta é pelo menos 1 livro por mês. Fora isso tenho zilhões de planejamento das coisas que quero fazer se me sobrar um tempo.

**Comment by kvnol** [comment]
Muito mais do que código, eu gosto de Deus e de música. Mais ainda quando os dois estão juntos: https://www.youtube.com/watch?v=lkwUvcO2lcU

**Comment by viniman** [comment]
Sem contar isso que o @kvnol disse, já toquei violão por um bom tempo na igreja, gosto muito, mas ainda contínuo frequentando, menos contínuo. Dei um tempo no violão e não estou muito mais envolvido, para cuidar mais de quem eu sou e fazer algumas decisões...

A propósito, belo violão, parabéns. Muito bom, gostei da música, voz maneira também @kvnol :D.

**Comment by valkervieira** [comment]
Quando minha filha está acordada eu passo praticamente o tempo todo brincando com ela, seja em casa fazendo bobice ou vendo TV (fiz ela gostar de studio ghibli)ou em parques, praças. As horas que restam gosto de jogar algum jogo do gênero RPG ou survival, atualmente to jogando meio quebradinho Don't Starve, Project Zomboid, a série Persona de PS1/PS2 e jogando pela enésima vez Stardew Valley. Também curto andar de bicicleta mas não faço no tempo livre pq geralmente significa menos tempo com a minha filha.

**Comment by ninetails** [comment]
Antigamente ia bastante a festivais de música eletrônica (deixar a barba, inclusive, foi intencional).

Acho que hoje acaba se resumindo a gaming tb. PS4 e comprei ultimamente um Switch pois sempre via uns streamings de gente jogando fases Trolls no Mario Maker 2. Jogava bastante Overwatch pra PC também...

<sup><sub><sup><sub>also uns anime e pá</sub></sup></sub></sup>

**Comment by felipe-heredia** [comment]
Fora do código (_no qual ainda estou aprendendo_), gosto muito de estudar sobre investimentos, como estudar empresas, ler livros etc. Dar algumas voltas de bicicleta pelo bairro, uns 9km por umas 3 vezes na semana. E também de cantar na igreja, amo cantar.

**Em resumo: O que gosto de fazer fora do código é exercitar-me e cantar.**

**Comment by LuanDc** [comment]
**Gosto muito de música eletrônica.** 

Geralmente procuro festivais pra frequentar, novas vertentes, sons e até já planejei de estudar produção musical como um hobby.

**Comment by romulosardinha** [comment]
Atualmente estou tentando adicionar um hábito de escrita. Além de ajudar nas organizações de minhas idéias, contribuo mais para comunidade.

**Comment by dionarodrigues** [comment]
É interessante saber que há um mundo além da tecnologia, rs. 

A leitura é um hábito que eu sempre tive e gosto muito. No entanto, morando em São Paulo e percorrendo longas distâncias dentro do transporte público, acabo deixando pra ler somente nesses horários de ida e volta do trabalho. Junto com isso acrescentei o hábito de ouvir podcasts (geralmente em inglês, assim aprendo algo novo e treino o inglês hehe).

Mas, desde o ano passado decidi quebrar a minha monótona rotina de ficar sempre fazendo as mesmas coisas e deicidi aprender violino, comecei a fazer terapia e a cuidar mais da minha saúde (em vários sentidos). Além disso, venho tentando assistir um filme ou série de vez em quando (algo que eu geralmente não costumo fazer).

Nesse momento de corona vírus venho estudando francês e encontrando meios alternativos de continuar cuidando do corpo e da saúde - e manter a sanidade hehe. :)

**Comment by kivervinicius** [comment]
@diogorodrigues qual o interesse em francês, algum motivo?

**Comment by dionarodrigues** [comment]
@kivervinicius Já fazia um tempo que vinha pesquisando sobre um novo idioma para aprender, uma vez que já considero o meu nível de inglês bom. Assim, levando em consideração o ranking das línguas mais faladas e utilizadas no mundo corporativo, acabei optando por francês (e também porque eu gosto muito da pronúncia).

Com isso, cito algumas vantagens:

- língua falada no mundo inteiro; é uma das línguas oficiais da ONU.
- fácil de aprender, já que assim como o português deriva do latim. Além de ter muitas palavras similares ao inglês, o que é ótimo para quem já sabe inglês.
- aumento das oportunidades de ir trabalhar no Canadá, que é um país que está sempre a procura de profissionais da área de tecnologia para trabalhar na província de Quebec.
- como eu já disse, acho a pronúncia muito bonita e elegante (rss)

**Comment by romulosardinha** [comment]
Neste confinamento ta foda conseguir atividades extras kkk to só trabalhando kkk
